src/DynamicDataTable.jsx

```diff
@@ -56,10 +56,10 @@ class DynamicDataTable extends Component {
     }
 
     className() {
-        const { onClick, onMouseUp, onMouseDown, hoverable } = this.props;
+        const { onClick, onMouseUp, onMouseDown, hoverable, className } = this.props;
 
         return classNames([
-            'table', 'table-striped',
+            ...className.split(' '),
```

Comment:
Why split the string here?

Reply:
I always try to disrupt existing code as little as possible; without splitting this would be evaluated as `"table table-striped"` rather than `"table", "table-striped"`

Reply:
While it's keeping it the same as the original code I think it makes sense just to pass a string. No need to add extra computation